### DISCUSSION TITLE
Rog Ally - Discoveries

### DISCUSSION THREAD

**Discussion by Valkirie** [seerge/g-helper#662] [General]
Maybe we should start a discussion to share finding on Rog Ally. Right now I'm struggling with one major issue.
- The ASUSTeK Computer Inc. ASUSTeK Computer Inc. device (vid = 0x0B05, pid = 0x1ABE) disconnect when device goes to sleep and never come back.

**Comment by seerge** [comment]
What is that device about ?

**Reply by Valkirie** [reply]
The one I listen to, to get custom buttons inputs.

**Reply by seerge** [reply]
Ah, i think i also experienced that with my app too on Ally :) did you try to send init command to it ?

**Reply by Valkirie** [reply]
No. Do you have that command somewhere?

**Reply by seerge** [reply]
https://github.com/seerge/g-helper/blob/e58a396cbdf5763a3e093ba9112cf818cb884131/app/AsusUSB.cs#L232 :)

That is aura init, but who knows ...
